### PR TITLE
Adding htwpasswd option to Vhost template

### PR DESCRIPTION
**Pull request by Eludendil**

### CODE REVIEW

**Comment by Eludendil** [comment]
Hi, just to understand .

How adding a new variable for ansible generate an nginx server error ?

**Comment by geerlingguy** [comment]
It looks like this change results in an invalid nginx config. From some of the failed tests:

```
TASK [role_under_test : Ensure nginx is started and enabled to start at boot.] ***
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Unable to start service nginx: Job for nginx.service failed because the control process exited with error code. See \"systemctl status nginx.service\" and \"journalctl -xe\" for details.\n"}
```

(And https://travis-ci.org/geerlingguy/ansible-role-nginx/jobs/249932686#L626, etc.).

Have you run the role with this change, and the role defaults, and seen if it runs correctly? The CI build just runs the job twice on a minimal linux install for many different OSes.

**Comment by Eludendil** [comment]
Thanks for you answer , i will dig into that. 
Do you think it's a good idea to map more variable for the role ?

File: templates/vhost.j2 [hunk 1]
```diff
@@ -32,7 +32,17 @@ server {
 {% endif %}
 {% if item.error_log is defined %}
     error_log {{ item.error_log }} error;
-{% endif %}
+{% endif %}}
```
Comment (LukeHandle) [inline templates/vhost.j2]: This should stay as `{% endif %}`?

**Pull request closed without merging**